apps/desktop/src/platform/preload.ts

```diff
@@ -71,6 +72,27 @@ const clipboard = {
   write: (message: ClipboardWriteMessage) => ipcRenderer.invoke("clipboard.write", message),
 };
 
+const sshagent = {
+  setKeys: (keys: { name: string; privateKey: string; uuid: string }[]): Promise<void> =>
+    ipcRenderer.invoke("sshagent.setkeys", keys),
+  signRequestResponse: async (id: number, accepted: boolean) => {
+    await ipcRenderer.invoke("sshagent.signrequestresponse", { id: id, accepted: accepted });
```

Comment:
suggestion: simplify 
```suggestion
    await ipcRenderer.invoke("sshagent.signrequestresponse", { id, accepted });
```